docs/geedocs/docsrc/Makefile

```diff
@@ -0,0 +1,28 @@
+# Minimal makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line.
+SPHINXOPTS    =
+SPHINXBUILD   = sphinx-build
+SPHINXPROJ    = GoogleEarthEnterpriseDocumentation
+SOURCEDIR     = .
+BUILDDIR      = build
+SPHINX2GITHUB = ./sphinxtogithub.py
+GETOPENGEEVER = ../../../earth_enterprise/src/scons/getversion.py
+
+OPENGEEVER := $(shell $(GETOPENGEEVER))
+
+# Put it first so that "make" without argument is like "make help".
+help:
+	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+
+.PHONY: help Makefile
+
+# Catch-all target: route all unknown targets to Sphinx using the new
+# "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
+%: Makefile
+	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+	@$(SPHINX2GITHUB) "$(BUILDDIR)/html"
+	rm -rf ../$(OPENGEEVER)/*
+	cp -r ./$(BUILDDIR)/html/* ../$(OPENGEEVER)
+	rm -rf ../$(OPENGEEVER)/sources
```

Comment:
What is this delete for?

Reply:
html directory also has sources directory which has a copy of all the rst files renamed as rst.txt. 
Some websites have the "show source" option and use them to display the files. I thought it is better to delete them as we are not planning to show source and it is a redundant copy of rst files.

Reply:
okay